apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/RifFilesEvent.java

```diff
@@ -16,21 +16,25 @@ public final class RifFilesEvent {
   private final Instant timestamp;
   /** The {@link RifFileEvent}s contained in this {@link RifFilesEvent}. */
   private final List<RifFileEvent> fileEvents;
+  /** boolean indicator denoting if data is synthetic (true). */
+  private final boolean syntheticData;
```

Comment:
Sometimes using a boolean for something like this can become troublesome in the future.  It's also less clear at the point of call when looking at the parameter list to know what `true` means.  Using an enum for the type of data could allow more nuanced treatment later without much refactoring.  For now it could just have values like Synthetic and Genuine.

Reply:
Yea, agreed but not my call; AC specified boolean with the backwards compatibility such that if the syntheticData is missing, then it defaults to false. So yes, I could do the same with an enum, but thats adds mores stuff to the XSD.